plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/IndicesRequestTests.java

```diff
@@ -178,7 +178,7 @@ public void testAnalyze() {
     }
 
     public void testIndex() {
-        String[] indexShardActions = new String[]{IndexAction.NAME, IndexAction.NAME + "[r]"};
+        String[] indexShardActions = new String[]{IndexAction.NAME + "[p]", IndexAction.NAME + "[r]"};
```

Comment:
Good catch, added back testing for the reroute phase for index, delete, refresh and flush actions.